fix(tuto): fix treafik tuto (#3552)
scaleway/docs-content: tutorials/traefik-v2-cert-manager/index.mdx

- A [Kubernetes Kapsule cluster](/containers/kubernetes/how-to/create-cluster/) with an ingress controller (Traefik 2)
- Downloaded the corresponding [kubeconfig](/containers/kubernetes/how-to/edit-cluster/) file and ensured the [kubectl](/containers/kubernetes/how-to/connect-cluster-kubectl/) is configured and working

### Deploying Traefik 2 using the Easy Deploy feature

1. Log into the [Scaleway console](https://console.scaleway.com/) and navigate to the **Kubernetes** section under **Containers**.
2. Click on your cluster name to view the cluster overview.
3. Go to the **Easy Deploy** tab.
4. Click **Deploy an Application**.

> 5. In the **Application Library**, search for **Traefik** and select **Traefik 2 Ingress**.

Reviewer: ```suggestion
5. Search for **Traefik**  in the **Application Library**, and select **Traefik 2 Ingress**.
```

6. Name the application `traefik` and set the namespace to `kube-system`.
7. Click **Deploy an application** to deploy Traefik 2.

> ### Creating a wildcard DNS Record

Reviewer 2: ```suggestion
### Creating a wildcard DNS record
```

        name: letsencrypt-prod
        kind: ClusterIssuer
    ```
3. Apply the certificate configuration:
    ```sh
    kubectl apply -f mycert.yaml
    ```
4. Verify the certificate creation:
    ```sh
    kubectl describe certificate teacoffee-cert
    ```

> ### Creating an HTTPS Ingress

Reviewer 2: ```suggestion
### Creating an HTTPS ingress
```

                    port:
                      number: 80
    ```
3. Apply the HTTPS ingress configuration:
    ```sh
    kubectl apply -f mysite.yaml
    ```
4. Test the HTTPS endpoint:
    ```sh
    curl -v https://teacoffee.mytest.com/tea
    ```

### Accessing the Traefik dashboard

1. Retrieve the name of the Traefik pod:
    ```sh
    kubectl get pods -n kube-system --selector "app.kubernetes.io/name=traefik" --output=name
    ```

>     This will output something like `pod/traefik-xxxxxxxxx-yyyyy`.

Reviewer 3: ```suggestion
    An output similar to the following should display: `pod/traefik-xxxxxxxxx-yyyyy`.
```
